src/core/simulation.jl

```diff
@@ -50,14 +45,13 @@ mutable struct Simulation
 
     function Simulation(;name::String,
                         steps::Int64,
-                        step_resolution::Dates.TimePeriod,
                         stages=Dict{String, Stage{AbstractOperationsProblem}}(),
                         stages_sequence=nothing,
                         simulation_folder::String,
                         verbose::Bool = false, kwargs...)
+    #step_resolution = IS.time_period_conversion(step_resolution)
```

Comment:
Stale?

Reply:
IS implemented methods to perform proper time period conversions, we should use those.